app/templates/suppliers/details.html

```diff
@@ -23,57 +23,48 @@
 {% block main_content %}
 {%
   with
-    heading = "Supplier details",
+    heading = "Company details",
     smaller = true
 %}
   {% include "toolkit/page-heading.html" %}
 {% endwith %}
 
-
-{{ summary.top_link('Edit', url_for('.edit_supplier')) }}
+{{ summary.heading("What buyers will see", id="what_buyers_will_see") }}
+{{ summary.top_link('Change', url_for('.edit_supplier')) }}
 {% call(item) summary.mapping_table(
-  caption='Supplier details',
+  caption='What buyers will see',
   field_headings=[
     'Label',
-    'Value'
+    'Value',
+    'Optional'
   ],
   field_headings_visible=False
 ) %}
   {% call summary.row() %}
     {{ summary.field_name('Contact name') }}
     {{ summary.text(supplier.contact.contactName) }}
+    {{ summary.text("") }}
   {% endcall %}
   {% call summary.row() %}
     {{ summary.field_name('Contact email') }}
     {{ summary.text(supplier.contact.email) }}
+    {{ summary.text("") }}
   {% endcall %}
   {% call summary.row() %}
     {{ summary.field_name('Contact phone number') }}
     {{ summary.text(supplier.contact.phoneNumber) }}
+    {{ summary.text("") }}
   {% endcall %}
   {% call summary.row() %}
-    {{ summary.field_name('Registered office address') }}
-    {% call summary.field() %}
-      {%
-        with
-        without_spacing = true,
-        postcode = supplier.contact.get("postcode"),
-        street_address = True,
-        street_address_line_1 = supplier.contact.get("address1"),
-        locality = supplier.contact.get("city")
-      %}
-        {% include "toolkit/contact-details.html" %}
-      {% endwith %}
-    {% endcall %}
-  {% endcall %}
-  {% call summary.row() %}
-    {{ summary.field_name('Supplier summary') }}
+    {{ summary.field_name('Summary') }}
     {{ summary.text(supplier.description) }}
+    {% call summary.field(action=True) %}
+      <font color="grey">Optional</font>
```

Comment:
We shouldn't be embedding font styles in our templates. I think if we really want grey text then it should become part of the toolkit pattern with a semantic meaning, e.g. `call summary.field(action=True, hint=True)`. 

For now I'd say go with normal text here, unless @laurenceberry is really strong on it being grey, in which case add the parameter to the toolkit.  Rather than grey it could go in brackets? i.e. `(Optional)` in the final column?

Reply:
OK, I agree the normal text styling is the best thing to do right now rather than update the frontend toolkit, that is a bit of a nice-to-have.